nilearn/plotting/find_cuts.py

```diff
@@ -269,3 +269,79 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     cut_coords.sort()
 
     return _transform_cut_coords(cut_coords, direction, affine)
+
+
+def find_parcellation_atlas_cut_coords(labels_img, background_label=0):
+
+    """ Grab coordinates of center of mass of 3D parcellation atlas
+
+    Parameters
+    ----------
+    labels_img: 3D Nifti1Image
+        A brain parcellation atlas.
+
+    background_label: number, optional
+        Label used in labels_img to represent background.
+
+    Returns
+    -------
+    labels_list: List
+        Label region values
+
+    coords: List
+        World coordinates for center of label regions
+    """
+
+    # grab data and affine
+    labels_img = check_niimg(labels_img)
+    labels_data = labels_img.get_data()
+    labels_affine = labels_img.get_affine()
+
+    # grab number of unique values in 3d image
+    unique_labels = set(np.unique(labels_data)) - set([background_label])
+
+    # grab center of mass from parcellations and dump into coords list
+    coord_list = []
+    label_list = []
+
+    for cur_label in unique_labels:
+        cur_img = labels_data == cur_label
+
+        #take the largest connected component
+        volume = np.asarray(cur_img)
+        labels, label_nb = ndimage.label(volume)
+        label_count = np.bincount(labels.ravel())
+        labels ==label_count.argmax()
+        label_count[0] = 0
+
+        # get parcellation center of mass
+        x,y,z = ndimage.center_of_mass(labels)
+
+        # dump label region and coordinates into a dictionary
+        label_list.append(cur_label)
+        coord_list.append((x,y,z))
+
+    # transform coordinates
+    coords = [coord_transform(i[0],i[1],i[2],labels_affine) for i in coord_list]
+
+    return label_list, np.array(coords)
+
+
+def find_probabilistic_atlas_cut_coords(label_img):
+    """ Grab coordinates of center probabolistic atlas 4D image
```

Comment:
typo. u got me. 
this func is to extract coords from 4d vols with fuzzy maps in each 3d (eg. msdl)